app/styles/layout/_modals.scss

```diff
@@ -164,6 +164,11 @@
   }
   .modal-wrapper {
     padding: 30px;
+    
+    form {
```

Comment:
Thanks for the PR! Don't know if grid is the right fit here. How about just:

```css
.auth-section {
  padding-bottom: 45px;
}
```

Reply:
Grid and flex is my preferred way of placing elements on the page)
Ok, i close this PR and create new.